docs/install.md

```diff
@@ -379,12 +379,11 @@ Follow the [Setup deployment directory](#3-setup-deployment-directory) instructi
 
     Uncomment `./hardware-configuration.nix` line by removing #.
 
-3. Create `hardware-configuration.nix`
+3. Create `hardware-configuration.nix` by copying the default configuration.
 
     ```
-    nano hardware-configuration.nix
+    cp /etc/nixos/hardware-configuration.nix .
```

Comment:
`cp` doesn't work because this is supposed to be run on the machine you're deploying from, but the `hardware-configuration.nix` should be taken from your freshly installed NixOS system.